### ISSUE TITLE
Should real-world models have realistic sizes, assuming meter units

### ISSUE THREAD

**Issue by jkieboom** [KhronosGroup/glTF-Sample-Models#309]
The glTF spec says that linear distances have meter units. It looks like some models (for example BoomBox) do not use meter units. It looks like it's off by maybe a factor 10.

**Comment by elalish** [comment]
A boom box for ants! Yes, this has been annoying me for awhile. Would be amazing if someone could do a pass over all the real-world-ish objects and make sure they have appropriate scale. I recall that damaged helmet is oddly huge as well.

**Comment by bghgary** [comment]
Yes, at least for the boom box and probably other models contributed by Microsoft, our team should fix it. I know we updated the FlightHelmet to be correctly sized relatively recently. We should try to fix them at the source instead of hand editing the glTF models though.

**Comment by javagl** [comment]
Related/duplicate: https://github.com/KhronosGroup/glTF-Sample-Models/issues/327

**Comment by DrX3D** [comment]
@bghgary : Has anything been done to address the original model sizes?

If no response to this comment, then the issue will be transferred to Sample Assets by 27 Nov.

**Comment by javagl** [comment]
This is a duplicate of https://github.com/KhronosGroup/glTF-Sample-Models/issues/327 (with this one being more "generic", and the other one containing a bit more specifc (and relevant) infos, e.g. about the actual sizes of the models).

So it could be OK to move either of them, and close the other (I _think_ that the mutual links remain intact even during a transfer, but in doubt, they can be added again after the transfer). 

The other question is that from the last comment of the other issue at https://github.com/KhronosGroup/glTF-Sample-Models/issues/327#issuecomment-1428800387 

> Is there a Khronos process for getting these issues resolved? Everyone on this thread is in agreement about what should be done, but no one is clearly an owner. Should Khronos put out an RFP for someone to actually fix/upkeep the models in this repo?

Yeah 😬 well. We can revisit this after the transfer. Scaling some models should be trivial with glTF-Transform.

**Comment by bghgary** [comment]
If we don't have to fix the source asset, then it should be relatively straight forward to fix assuming we can figure out the proper physical size to target.

**Comment by bghgary** [comment]
Sorry, I didn't answer the question.

> @bghgary : Has anything been done to address the original model sizes?

No, we have not done anything to address the model sizes.
